src/CMakeLists.txt

```diff
@@ -759,9 +771,16 @@ ELSE(APPLE OR WIN32)
 ENDIF(APPLE OR WIN32)
 
 OPTION(VISIT_THREAD "Build VisIt with thread support." OFF)
-OPTION(VISIT_BUILD_AVTEXAMPLES  "Enable building of avtexamples" ON)
-OPTION(VISIT_BUILD_DIAGNOSTICS  "Enable building of diagnostices" ON)
 
+# some options that when turned off reduce the build targets
+OPTION(VISIT_ENABLE_ANNOTATION_TOOLS "Enable generation of annotation tools (text2polys, time_annotation)" ON)
+OPTION(VISIT_ENABLE_DATAGEN "Enable generation of sample data files" ON)
+OPTION(VISIT_ENABLE_DATA_MANUAL_EXAMPLES "Build Getting Data Into VisIt examples" OFF)
+OPTION(VISIT_ENABLE_DIAGNOSTICS  "Enable building of diagnostices" ON)
+OPTION(VISIT_ENABLE_EXAMPLES  "Enable building of examples" ON)
```

Comment:
The `AVT` got dropped in the rename. Without it, the option seems a tad too generic. Can we cname this `VISIT_ENABLE_AVT_EXAMPLES`

Reply:
I added viewerproxy example under the umbrella of VISIT_ENABLE_EXAMPLES, that's why I changed the name.